hw_3.js

```diff
@@ -0,0 +1,70 @@
+
+// Task 1
+let a1 = 'true',
+    a2 = false,
+    a3 = 17,
+    a4 = undefined,
+    a5 = null;
+console.log(typeof(a1) + ',', typeof(a2) + ',', typeof(a3) + ',', typeof(a4) + ',', typeof(a5));
+
+// Task 2
+let height = 15, width = 20;
+if (height > width) {console.log('Высота больше ширины')}
+else console.log('Ширина больше высоты');
+
+// Task 3
+for (let i = 1; i <= 20; i++){
+    if (i % 3 === 0) console.log(i)}
+
+// Task 4
+let key = true,
+    documents = true,
+    pen = true,
+    apple = false,
+    orange = true,
+    shouldGoToWork;
+if ((key && documents && pen) && (apple || orange)) {
+    shouldGoToWork = 'На работу иду'; console.log(shouldGoToWork)}
+else {shouldGoToWork = 'Никуда не пойду'; console.log(shouldGoToWork)};
+
+// Task 5
+let myNumber = +prompt('Введите число:');
+if ((myNumber % 3 === 0) && (myNumber % 5 === 0)) {alert('FizBuz')}
+else if (myNumber % 5 === 0) {alert('Fiz')}
+else if (myNumber % 3 === 0) {alert('Buz')};
```

Comment:
You can improve it for non number values, just to add ELSE { console.log('it is not a number') }